src/tactic/ui/tools/ingest_wdg.py

```diff
@@ -1041,8 +1347,8 @@ def execute(my):
             return date.split(" ")[0]
         """
 
-        if not SearchType.column_exists(search_type, "name"):
-            raise TacticException('The Ingestion puts the file name into the name column which is the minimal requirement. Please first create a "name" column for this sType.')
+        if not SearchType.column_exists(search_type, column):
+            raise TacticException('The Ingestion puts the file name into the name column which is the minimal requirement. Please first create a "%s" column for this sType.' % column)
```

Comment:
In the error message, name column should be %s column and u can substitute it with the new column variable